go/vt/vtgate/vtgateservice/vtgateservice_testing/mock_vtgateservice.go

```diff
@@ -1,25 +1,12 @@
-/*
-Copyright 2017 Google Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreedto in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// Automatically generated by MockGen. DO NOT EDIT!
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
All these mock regenerations are not related to locking, right? Why are they here?

Reply:
Because I needed to regenerate the mocks, and they all come as one. I used a different commit for the others, so it's clearer.

Reply:
So you can regenerate the mocks with all the changes unrelated to this PR as a separate PR, merge them, and then this PR will contain only changes in the mocks that are related to it, right?